contracts/wveOLAS.sol

```diff
@@ -243,21 +261,54 @@ contract wveOLAS {
         vPower = IVEOLAS(ve).getPastTotalSupply(blockNumber);
     }
 
+    /// @dev Gets the total number of supply points.
+    /// @return numPoints Number of supply points.
+    function totalNumPoints() external view returns (uint256 numPoints) {
+        numPoints = IVEOLAS(ve).totalNumPoints();
+    }
+
```

Comment:
Why now we are doing this for `totalNumPoints()` and not for `mapSupplyPoints`?

Reply:
They are just very custom, but ok let's add them as well